### PR TITLE
Update sasl-irssi.shtml and cap_sasl.pl

### LINKED ISSUE #1: sasl-irssi
I think that the [sasl-irssi](https://freenode.net/sasl/sasl-irssi.shtml) page should link to scripts.irssi.org version at https://github.com/irssi/scripts.irssi.org/blob/gh-pages/scripts/cap_sasl.pl instead of linking to (possibly old) copy hosted by freenode.

> ECDSA-NIST256P-CHALLENGE is a certificate-based challenge authentication supported by newer versions of services. It requires additional code and components not supported by this script, but you can find an alternative version with Atheme's ecdsatool.

again links to Atheme's ecdsatool which is not the full script (https://github.com/kaniini/ecdsatool/pull/4).

### PR DESCRIPTION
**Pull request by Mikaela**
Closes #1 

I am not sure on removing the script entirely yet as some people might always use it instead of scripts.irssi.org version.

### CODE REVIEW

File: sasl/sasl-irssi.shtml [hunk 1]
```diff
@@ -3,10 +3,10 @@
 <!--#set var="content_title" value="Configuring SASL for irssi" -->
 <!--#include virtual="${VIRTROOT}include/header-mainlogos.shtml" -->
 
-<p>This script, by Michael Tharp and Jilles Tjoelker, comes from <a href="http://www.stack.nl/~jilles/irc/">http://www.stack.nl/~jilles/irc/</a>. Authentication information is stored in ~/.irssi/sasl.auth.</p>
+<p>This script, by Michael Tharp (gxti), Jilles Tjoelker (jilles) and Mantas Mikulėnas (grawity), comes from <a href="http://scripts.irssi.org/">http://scripts.irssi.org/</a>. Authentication information is stored in ~/.irssi/sasl.auth.</p>
 
 <ol>
-	<li><p>Copy the script, <a href="cap_sasl.pl">cap_sasl.pl</a>, into your ~/.irssi/scripts/autorun directory or from wherever irssi loads startup scripts.</p></li>
+	<li><p>Copy the script, <a href="cap_sasl.pl">https://github.com/irssi/scripts.irssi.org/raw/gh-pages/scripts/cap_sasl.pl</a>, into your ~/.irssi/scripts/autorun directory or from wherever irssi loads startup scripts.</p></li>
```
Comment (grawity) [inline sasl/sasl-irssi.shtml]: This should change the href instead, I think?

Comment (Mikaela) [inline sasl/sasl-irssi.shtml]: Fixed, thanks

**Comment by Mikaela** [comment]
I remembered that this exists.

Is there something wrong with this pull request that I should fix (what?) or doesn't anyone just ever check pull requests here?

**Comment by kjayne** [comment]
This update has been in the to-do queue; and it's done now. We're finally out of the business of hosting cap_sasl.pl and it lives on scripts.irssi.org, like it should.

**Pull request closed without merging**